conf/local.conf.js

```diff
@@ -52,7 +52,12 @@ exports.config = {
   },
 
   // Code to stop browserstack local after end of test
-  onComplete: function (exitCode, config, capabilities, results) {
-    exports.bs_local.stop();
-  },
+  onComplete: function (capabilties, specs) { 
```

Comment:
Why the change in the arguments. I can see neither the previous arguments were being used nor the new ones.

How the user sessions will be impacted?

Is it a semver-major changes?

Reply:
It is on our documentation:https://www.browserstack.com/automate/webdriverio#running-local-test